core/src/main/scala/scalaz/ListT.scala

```diff
@@ -102,7 +102,13 @@ sealed abstract class ListTInstances extends ListTInstances1 {
     new ListTHoist {}
 }
 
-object ListT extends ListTInstances {
+trait ListTFunctions {
+  def listT[M[_]] = new (λ[α => M[List[α]]] ~> ListT[M, ?]) {
```

Comment:
very, very minor nit, but could the type sig be added, and the impl put on the next line?